src/parse/rules/go_rules.build_defs

```diff
@@ -658,14 +660,24 @@ def _go_binary_cmds(static=False, ldflags=''):
         flags = ''
 
     return {
-        'dbg': '%s; %s %s $SRCS' % (_LINK_PKGS_CMD, _link_cmd, flags),
-        'opt': '%s; %s %s -s -w $SRCS' % (_LINK_PKGS_CMD, _link_cmd, flags),
+        'dbg': '%s && %s %s $SRCS' % (prefix, _link_cmd, flags),
+        'opt': '%s && %s %s -s -w $SRCS' % (prefix, _link_cmd, flags),
     }, {
         'go': [CONFIG.GO_TOOL],
         'ld': [CONFIG.LD_TOOL if CONFIG.LINK_WITH_LD_TOOL else CONFIG.CC_TOOL],
     }
 
 
+def _go_import_path_cmd(import_path):
+    """Returns a partial command which is used for setting up the Go import path."""
+    if not import_path:
+        return ''
+    elif '/' in import_path:
```

Comment:
do you need to guard against an import path starting with `/` ?

Reply:
Probably should, yeah; it wouldn't make sense as a Go import path but someone might do it by accident.